src/Compilers/VisualBasic/Portable/Scanner/ScannerInterpolatedString.vb

```diff
@@ -112,22 +111,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         End Function
 
         Private Function IsInterpolatedStringPunctuation(Optional offset As Integer = 0) As Boolean
-            If Not CanGet(offset) Then Return False
-
-            Dim c = Peek(offset)
-
+            Dim c, c1 As Char
+            If Not TryPeek(offset, c) Then Return False
+            Dim ok = TryPeek(offset + 1, c1)
```

Comment:
Consider inlining TryPeek(offset + 1, c1), otherwise we might call it in cases when the call is not needed.